content/codespaces/getting-started-with-codespaces/getting-started-with-your-python-project-in-codespaces.md

```diff
@@ -0,0 +1,221 @@
+  
+  ![Open with Codespaces button](/assets/images/help/codespaces/open-with-codespaces-button.png) 
+
+2. To create a new codespace, click {% octicon "plus" aria-label="The plus icon" %} **New codespace**.
+  ![New codespace button](/assets/images/help/codespaces/new-codespace-button.png)
+
+When you create a codespace, your project is created on a remote VM that is dedicated to you. By default, the container for your codespace has many languages and runtimes including Node.js, JavaScript, Typescript, nvm, npm, and yarn. It also includes a common set of tools like git, wget, rsync, openssh, and nano. 
+
+You can customize your codespace by adjusting the amount of vCPUs and RAM], [adding dotfiles to personalize your environment](/codespaces/setting-up-your-codespace/personalizing-codespaces-for-your-account), or by modifying the tools and scripts installed. 
+
+{% data variables.product.prodname_codespaces %} uses a file called `devcontainer.json` to store configurations. On launch {% data variables.product.prodname_codespaces %} uses the file to install any tools, dependencies, or other set up that might be needed for the project. For more information, see "[Configuring Codespaces for your project](/codespaces/setting-up-your-codespace/configuring-codespaces-for-your-project)."
+
+The next section shows you how to modify your tools by adding a dev container.
+
+### Step 2: Add a dev container to your codespace from a template  
+
+The default codespaces container will support running Python projects like [python-docs-hello-world](https://github.com/2percentsilk/python-docs-hello-world) out of the box. By setting up a custom container you can customize the tools and scripts that run as part of codespace creation and ensure a fully reproducible environment for all {% data variables.product.prodname_codespaces %} users in your repository.
+
+To set up your project with a custom container, you will need to use a `devcontainer.json` file to define the environment. In {% data variables.product.prodname_codespaces %} you can add this either from a template or you can create your own. For more information on dev containers, see [Configuring your codespace](/codespaces/setting-up-your-codespace/configuring-codespaces-for-your-project). 
+
+This example guides you through adding a `devcontainer.json` file from a template.
+
+1. Access the command palette (`shift command P` / `shift control P`), then start typing "dev container". Click **Codespaces: Add Development Container Configuration Files...**
+  !["Codespaces: Add Development Container Configuration Files..." in the command palette](/assets/images/help/codespaces/add-prebuilt-container-command.png)
+3. For this example, click **Python 3**. In practice, you could select any container that’s specific to Python or a combination of tools such as Python 3 and PostgresSQL.
```

Comment:
```suggestion
3. For this example, click **Python 3**. If you need additional features you can select any container that’s specific to Python or a combination of tools such as Python 3 and PostgresSQL.
```

Avoiding the phrase "in practice" as it makes it feel like you're having them walk through something they wouldn't normally do. Love that you have the example at the end, so just rolling straight into it.